extensions/authentication-passport/src/__tests__/acceptance/authentication-with-passport-strategy-adapter.acceptance.ts

```diff
@@ -29,62 +31,154 @@ import {
+      users.list.Joseph.profile.username + ':' + users.list.Joseph.password;
     const hash = Buffer.from(credential).toString('base64');
     await client
       .get('/whoAmI')
       .set('Authorization', 'Basic ' + hash)
-      .expect(users.list.joe.profile[securityId]);
+      .expect(users.list.Joseph.profile.id);
   });
 
   it('returns error for invalid credentials', async () => {
+    await remainingSetup(ScenarioEnum.WithoutProviders);
     const client = whenIMakeRequestTo(server);
-    const credential = users.list.Simpson.profile[securityId] + ':' + 'invalid';
+    const credential = users.list.Simon.profile.username + ':' + 'invalid';
     const hash = Buffer.from(credential).toString('base64');
     await client
       .get('/whoAmI')
       .set('Authorization', 'Basic ' + hash)
       .expect(401);
   });
 
-  it('allows anonymous requests to methods with no decorator', async () => {
+  it(`allows anonymous requests to methods with no 'authenticate' decorator`, async () => {
     class InfoController {
       @get('/status')
       status() {
         return {running: true};
       }
     }
 
-    app.controller(InfoController);
+    app.controller(InfoController); //overriding 'beforeEach(givenControllerInApp);' above
+    await remainingSetup(ScenarioEnum.WithoutProviders);
     await whenIMakeRequestTo(server)
       .get('/status')
       .expect(200, {running: true});
   });
 
-  // FIXME: In a real database the column/field won't be a symbol
+  it('Using provider - authenticates successfully for correct credentials', async () => {
+    await remainingSetup(ScenarioEnum.WithProviders);
+    const client = whenIMakeRequestTo(server);
+    const credential =
+      users.list.Joseph.profile.username + ':' + users.list.Joseph.password;
+    const hash = Buffer.from(credential).toString('base64');
+    await client
+      .get('/whoAmI')
+      .set('Authorization', 'Basic ' + hash)
+      .expect(users.list.Joseph.profile.id);
+  });
+
+  it('Using provider - returns error for invalid credentials', async () => {
+    await remainingSetup(ScenarioEnum.WithProviders);
+    const client = whenIMakeRequestTo(server);
+    const credential = users.list.Simon.profile.username + ':' + 'invalid';
+    const hash = Buffer.from(credential).toString('base64');
+    await client
+      .get('/whoAmI')
+      .set('Authorization', 'Basic ' + hash)
+      .expect(401);
+  });
+
+  it(`Using provider - allows anonymous requests to methods with no 'authenticate' decorator`, async () => {
+    class InfoController {
+      @get('/status')
+      status() {
+        return {running: true};
+      }
+    }
+
+    app.controller(InfoController); //overriding 'beforeEach(givenControllerInApp);' above
+    await remainingSetup(ScenarioEnum.WithProviders);
+    await whenIMakeRequestTo(server)
+      .get('/status')
+      .expect(200, {running: true});
+  });
+
+  async function givenAnApp() {
+    app = new Application();
```

Comment:
creating an instance of [RestApplication](https://github.com/strongloop/loopback-next/blob/master/packages/rest/src/rest.application.ts#L40) should be good, then you don't need mount the RestComponent.